views/js/qtiCreator/widgets/component/minMax/minMax.js

```diff
@@ -350,7 +350,8 @@ define([
                         }
                     } else if (minValue === 0 && maxValue > 0) {
                         this.enableField(fields.min, 1);
-                        this.$component[0].querySelector('input[name=minChoices-toggler]').checked = true;
+                        const input = this.$component[0].querySelector('input[name=minChoices-toggler]') || this.$component[0].querySelector('input[name=minAssociations]');
```

Comment:
Match interaction don't have checkbox `toggler: false`
No need to select input it is already stored in controls
```
                        } else if (config[fields.min].toggler === true && minValue === 0 && maxValue > 0) {
                            this.enableField(fields.min, 1);
                            controls.min.input.checked = true;
                        }
```